modules/Kconfig.mbedtls

```diff
@@ -3,8 +3,17 @@
 # Copyright (c) 2016 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
 
+config MBEDTLS_PROMPTLESS
+	bool
+	help
+	  Symbol to disable the prompt for MBEDTLS selection.
+	  This symbol may be used internally in a Kconfig tree to hide the
+	  mbedTLS menu prompt and instead handle the selection of MBEDTLS from
+	  dependent sub-configurations and thus preven stuck symbol behavior.
+
+
 menuconfig MBEDTLS
-	bool "mbedTLS Support"
+	bool "mbedTLS Support" if !MBEDTLS_PROMPTLESS
```

Comment:
I've tried this with `echo_server`/`echo_client` samples and I can still see this entry even though `MBEDTLS_PROMPTLESS` is set to `y` (the default configuration). What am I missing?

Reply:
The fact that `mbedTLS` is a menuconfig and not a config.
Thus, when another Kconfig selects the promptless menuconfig, you will see the symbol in order to access the rest of the menu.

But if `MBEDTLS=n` and  `MBEDTLS_PROMPTLESS=y` then you will not see the menu and cannot enable it.

If `MBEDTLS_PROMPTLESS=n` you can of course both see, configure and enter the menu.